Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -29,6 +29,7 @@
         - HolofanProjector
         - Multitool
         - AppraisalTool
+        - FireExtinguisherMini
```

Comment:
whitelist requires tag, not an entity id, so you should make a FireExtinguisherMini tag

Reply:
so that would require it being in the fire_extinguisher.yml?

Reply:
in the fire_extinguisher.yml?